plugins/e2e-harness/skills/e2e-setup/SKILL.md

```
@@ -0,0 +1,98 @@
+---
+name: e2e-setup
+description: Onboard a full Playwright E2E test harness in the current project — verify/install Playwright, generate the official planner/generator/healer agents via npx playwright init-agents --loop=claude, scaffold auth separation (storageState + setup project), network route mocking, an E2E operating SSOT doc, and a gated GitHub Actions CI workflow with trace artifacts and PR-failure comments. Use when the user asks to set up E2E, add Playwright AI agents, bootstrap end-to-end testing, or wire E2E into CI. Never overwrites an existing playwright.config (merge proposal + backup). Degrades gracefully when Playwright is absent. Run from the user's project root, not this marketplace repo.
+allowed-tools: Read Write Edit Bash Glob Grep AskUserQuestion
+---
+
+# E2E Setup — full Playwright harness onboarding
+
+Stand up Playwright's official AI test harness (planner -> generator -> healer) plus the surrounding engineering (auth separation, deterministic mocking, an E2E SSOT doc, gated CI). The harness is the point: a test run is a sensor, a test file is a spec, and the three agents form a self-improving loop. This skill only does **setup + orchestration + CI + integration** — it does not re-implement the agents (Playwright generates them natively).
+
+> Bundled templates live at `${CLAUDE_PLUGIN_ROOT}/assets/` under Claude Code; under Codex 0.135 the same `assets/` dir sits next to `skills/` in the plugin cache. The three template files are `playwright-ci.yml`, `e2e-guidelines.template.md`, `route-mock.scaffold.ts`.
+
```

markdownlint-cli2 (0.22.1)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)